src/content/docs/developer-tools/your-apis/test-token-from-kinde.mdx

```diff
@@ -1,6 +1,6 @@
 ---
 page_id: be39d3bb-a83d-4d21-8941-8f1c8363c3bb
-title: Get a user access token to test your APIs (Fast method)
+title: Get a M2M token to test your APIs
```

Comment:
@clairekinde11 in my head it feels like `an` not `a` when I read it out loud `Get an M2M token to test your APIs`

Reply:
I know but ...